### DISCUSSION TITLE
Feature request: it would be nice to have a better way to view the README section

### DISCUSSION THREAD

**Discussion by tdeenv** [community/community#58998] [Repositories]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

At the moment the README section is tiny and in the sidebar where it isn't easy to view or to fit much info in. I think it would be nice to have a larger open text section somewhere kind of like how Notion lets you have a description above your table.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @thaherxyz ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
